src/backtrace/dbghelp64.rs

```diff
@@ -1,19 +1,10 @@
-//! Backtrace strategy for MSVC platforms.
+//! Backtrace strategy for MSVC `x86_64` and `aarch64` platforms.
```

Comment:
Apparently it's used also for gnu and gnullvm: https://github.com/rust-lang/backtrace-rs/blob/e15130618237eb3e2d4b622549f9647b4c1d9ca3/src/backtrace/mod.rs#L173-L186

Also confirmed here: https://github.com/msys2/MINGW-packages/pull/20651#issuecomment-2067510598

Reply:
Hey it was like that when I got here 😅

But yes, that should just say Windows. I guess it was once MSVC only.

Reply:
Sure, I just wanted to point out a preexisting issue.